src/collections/_documentation/integrations/bitbucket.md

```diff
@@ -1,23 +1,51 @@
 ---
 title: Bitbucket
-sidebar_order: 1
+sidebar_order: 2
 ---
+Sentry’s new Bitbucket integration has the following features: commit tracking and issue management. You can now use the data from your commits to Bitbucket to help you find and fix bugs faster.
 
-Sentry integrates with both Bitbucket Issues and Pipelines.
+Note: This feature is only available for early adopters. To become an early adopter, navigate to Organization Settings > General Settings, and enable 'Early Adopter.'
 
-## Set up Bitbucket Issue creation in Sentry
+## Configure Bitbucket
 
-1.  Go to the project settings page in Sentry that you’d like to link with Bitbucket
-2.  Click All Integrations, find the Bitbucket integration in the list, and click configure
-3.  Click Enable Plugin
-4.  Fill in the required information and save
-5.  The option to create Bitbucket issues will be displayed from your Sentry issue pages
 
-## Set up release tracking with Bitbucket Pipelines
+1. In Sentry, navigate to Organization Settings > Integrations. *Note: only users with Owner and Manager permissions will have access to this page.*
```

Comment:
Not critical, but if you make this list with all `1.` markdown will automatically number them, which makes for cleaner diffs if items have to be added/removed